content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -0,0 +1,398 @@
+| Serial number                   | Lot         | (21)                         | Alpha-numeric name | Serial number      |
+|                                 |             | ([!"%-/0-9:-?A-Z_a-z]{0,20}) |                    |                    |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Packaging date (YYMMDD)         | Packaging   | (13)(\\d{6})                 | Date               | Pack date          |
+|                                 | Date        |                              |                    |                    |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Best before date (YYMMDD)       | Best before | (15)(\\d{6})                 | Date               | Best before date   |
+|                                 | Date        |                              |                    |                    |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Expiration date (YYMMDD)        | Expiration  | (17)(\\d{6})                 | Date               | Expiry date        |
+|                                 | Date        |                              |                    |                    |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Variable count of items         | Quantity    | (30)(\\d{0,8})               | Measure            | UoM: Units         |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Count of trade items            | Quantity    | (37)(\\d{0,8})               | Measure            | Qty in units for   |
+|                                 |             |                              |                    | containers (AI 02) |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Net weight: kilograms (kg)      | Quantity    | (310[0-5])(\\d{6})           | Measure            | Qty in kg          |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Length in meters (m)            | Quantity    | (311[0-5])(\\d{6})           | Measure            | Qty in m           |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Net volume: liters (L)          | Quantity    | (315[0-5])(\\d{6})           | Measure            | Qty in L           |
++---------------------------------+-------------+------------------------------+--------------------+--------------------+
+| Net volume: cubic meters (m^3)  | Quantity    | (316[0-5])(\\d{6})           | Measure            | Qty in m^3         |
```

Comment:
If there's a way to format the exponents in LaTeX (so they look like exponents instead of `m^3`) that would be great!

Reply:
Did it :heart: